meetings/2020-02-25.md

```diff
@@ -0,0 +1,32 @@
+## 2020-02-25 Summit Planning Meeting
+
+**Present:** Robin, Rachel, Ruy, Jory, Manil, Wes, Waleed
+
+**Agenda**
+
+
+
+1. **Meta Issues**
+    1. Use project board / milestones for tracking deadlines, proposals, tasks?
+     * Yes, +1
+    2. Update README
+     * Waleed volunteers to work on PR
+2. **Venue & Date Updates**
+    3. June 25-26 - Capital Factory on HOLD!! Woo!
+3. **Registration**
+    * Registration is open - $15 cost is an add on or stand-alone registration
```

Comment:
It this a straight cost or a fee you pay if you register and then don't show up?

Reply:
A straight cost you pay to register for the summit. AFAIK this is non-refundable.

Reply:
This is a departure from past summits. Is there any minutes capturing discussion on this front? It's not a concern for me personally, but would be interested in the discussion that motivated the change.

Reply:
We discussed a lot about this and compared it with the previous "no show" fee of 50 dollars. We all agreed that a token fee could be better and at the same time it helps preventing no-shows while helping the summit in paying for itself. This was inspired by what Electron do for their summits.

The reasoning behind is that most of us can expense this and it will be possible to get this covered by the travel fund if somebody applies for it.

Reply:
@mcollina sounds good.  Given that most people should be able to expense either through the travel fund or internal funding should not be a concern.